lib/logstash/outputs/elasticsearch.rb

```diff
@@ -158,7 +158,7 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # control over template creation, (e.g. creating indices dynamically based on
   # field names) you should set `manage_template` to false and use the REST
   # API to apply your templates manually.
-  config :manage_template, :validate => :boolean, :default => true
+  config :manage_template, :validate => :boolean # :default => true unless data_stream_config?
```

Comment:
I would add this information as part of the comment instead of commenting part of the line
```suggestion
# Is false for data streams else it's true
  config :manage_template, :validate => :boolean
```